spec/bowling_scorecard_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'bowling_scorecard'
+
+describe Scorecard do
+  let(:scorecard) { described_class.new}
+
+  describe 'bowls' do
+    it 'lets a player make bowls' do
+      expect(scorecard).to respond_to(:bowl).with(1).argument
+    end
+  end
+
+context 'score' do
+  describe 'score' do
+    it 'calculates score for finished game' do
+      expect(scorecard).to respond_to(:bowls)
+    end
+  
+    it 'returns the score' do
+      20.times{ scorecard.bowl(2) }
+      expect(scorecard.score).to eq(40)
+    end
+    
+    it 'returns 0 for a gutter game' do
+      20.times{ scorecard.bowl(0) }
+      expect(scorecard.score).to eq(0)
+    end
+
+   context 'when there is a spare' do
```

Comment:
The syntax for these tests (#.times do) do well to overcome my reservations at the length of the tests on my own version